### DISCUSSION TITLE
Should we try to switch using mdn/data in a minifier?

### DISCUSSION THREAD

**Discussion by ludofischer** [mdn/mdn-community#716] [Platform] [answered]
Hello, I am one of the maintainers of the [cssnano](https://www.npmjs.com/package/cssnano) package, which is a css minifier. Currently it uses the `css/properties.json` file of the [mdn data](https://github.com/mdn/data) repository to dtermine whether to replace a property value with the `initial` keyword or vice-versa. Does the deprecation notice on the `mdn/data` package mean that `css/properties.json` is soon not going to be update anymore and we should try to work with the data in the `ed`  directory of  [webref](https://github.com/w3c/webref)?

**Comment by Josh-Cena** [comment, marked as answer]
Since you haven't got an answer so far I will give what I know: yes, you should consider switching your webref if it works. mdn/data is not unmaintained but it will be only maintained as much as necessary for our content. OTOH webref is guaranteed to be up to date. A large part of MDN is already powered by webref and we are considering whether mdn/data should just rebundle webref (I forgot which specific parts of it).

**Reply by ludofischer** [reply]
Thank you for your reply, webref is just slightly less convenient because we need the initial value for every CSS property and CSS properties are spread across multiple files in webref.

**Comment by skyclouds2001** [comment]
see https://github.com/mdn/mdn/issues/166, this seems not to happen recently
